akka-persistence-typed/src/main/scala/akka/persistence/typed/scaladsl/PersistentActor.scala

```diff
@@ -33,12 +35,14 @@ object PersistentActor {
     persistenceIdFromActorName: String ⇒ String,
     initialState:               State,
     commandHandler:             CommandHandler[Command, Event, State],
-    eventHandler:               (State, Event) ⇒ State): PersistentBehavior[Command, Event, State] =
+    eventHandler:               (State, Event) ⇒ State,
+    taggingFunction:            Event ⇒ Seq[String]                   = (_: Event) ⇒ Seq.empty): PersistentBehavior[Command, Event, State] =
```

Comment:
An alternative is to add it to `PersistentBehavior`, similar to `onRecoveryCompleted`. That might make more sense since it's an optional feature.

Reply:
use `Set` instead of `Seq`